custodial-copy-reconciler/src/test/scala/uk/gov/nationalarchives/reconciler/OcflServiceSpec.scala

```diff
@@ -0,0 +1,79 @@
+package uk.gov.nationalarchives.reconciler
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import io.ocfl.api.model.{ObjectVersionId, VersionInfo}
+import org.apache.commons.codec.digest.DigestUtils
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers.*
+import uk.gov.nationalarchives.reconciler.Main.Config
+import uk.gov.nationalarchives.utils.Utils.*
+
+import java.nio.file.{Files, Path}
+import java.util.UUID
+
+class OcflServiceSpec extends AnyFlatSpec {
+
+  "getAllObjectFiles" should "return all files in the repository" in {
+    val (repoDir, workDir) = (Files.createTempDirectory("repo").toString, Files.createTempDirectory("work").toString)
+    val repository = createOcflRepository(repoDir, workDir)
+    val id = UUID.randomUUID
+
+    case class IdPath(preservationId: UUID, path: Path)
+
+    val idsPaths = List.fill(100)(UUID.randomUUID).map { preservationId =>
+      val preservationTestFile = Files.createTempFile(preservationId.toString, "file")
+      val path = Files.write(preservationTestFile, preservationId.toString.getBytes)
+      IdPath(preservationId, path)
+    }
+    repository.updateObject(
+      ObjectVersionId.head(id.toString),
+      new VersionInfo(),
+      updater => {
+        idsPaths.map { idPath =>
+          updater
+            .addPath(idPath.path, s"$id/Preservation_1/${idPath.preservationId}")
+        }
+      }
+    )
+
+    val config = Config("", "test-database", 1, repoDir, workDir)
+    val allFiles = OcflService[IO](config).getAllObjectFiles.compile.toList.unsafeRunSync()
+    allFiles.length should equal(100)
+
+    idsPaths.map(_.preservationId).map { preservationId =>
+      val file = allFiles.find(_.id == preservationId).get
+      file.parent.get should equal(id)
+      file.sha256Checksum.get should equal(DigestUtils.sha256Hex(preservationId.toString))
+    }
+  }
+
+  "getAllObjectFiles" should "return access copies" in {
```

Comment:
1. Should we add capitalise and/or add quotes around "Access"?
2. Should we make this "return 'Preservation' and 'Access' copies"?

Reply:
Yeah ok.